printing-dagservice.go

```diff
@@ -0,0 +1,63 @@
+package dex
```

Comment:
Right direction, but I think since this is ipfs-cluster's DAG service, this should be part of ipfs-cluster codebase, which will import `go-ipld-importers` or whatever we call it as needed.

Reply:
ups sorry this one's closed